src/styles/partials/_mixins.scss

```diff
@@ -26,3 +26,16 @@
   border: none;
   background: none;
 }
+
+@mixin helsinki-logo($_color) {
```

Comment:
should be in its own file, maybe a variable.

Reply:
Not sure where you want to move this.